packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -196,9 +196,6 @@ def _get_create_batch_tx(
         """Create a batch of items."""
         # create the items
         self.nonce += 1
-        nonce = ledger_api.api.eth.getTransactionCount(deployer_address)
-        if self.nonce != nonce:
-            self.nonce = nonce
```

Comment:
Removed the check here. Since we are keeping track of the nonce. Actually creates a problem because the nonce is always less than the supposed nonce.

Reply:
Well it makes sense to check the nonce. Just this particular check does not make sense. Leave it for me to finish.